components/header/menu/index.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+
+import { TOOLS } from '../constants';
+import cx from 'classnames';
+import Icon from 'components/icon';
+
+const HeaderMenu = () => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <div className="row justify-content-start justify-content-sm-center">
+      <button className="btn btn-link p-0" onClick={() => setIsOpen(true)}>
```

Comment:
By default, buttons are submit buttons.

```suggestion
      <button type="button" className="btn btn-link p-0" onClick={() => setIsOpen(true)}>
```

Reply:
Could you also improve the focus style? The other buttons and inputs have an focus ring.